users/pyproject.toml

```diff
@@ -19,10 +19,10 @@ email-validator = "^1.1.3"
 apscheduler = "^3.9.1"
 pyjwt = "^2.6.0"
 badgerdoc-storage = { path = "../lib/badgerdoc_storage" }
+black = {path = "../lib/tenants"}
```

Comment:
Could you elaborate on this? It should be tenants, not black, right?

Reply:
Yeah, it's weird dependency, removed at all. Thank you for catching it.